docs/learn/learn-staking-miner.md

```diff
@@ -48,7 +51,28 @@ order to submit solutions to the NPoS election. When the correct time comes, it
 solution and submits it to the chain. The default miner algorithm is sequential Phragmén with a
 configurable number of balancing iterations that improve the score.
 
-![NPoS election optimization](../assets/staking-miner/NPoS-election-optimization.png)
+## Deposit and reward mechanics
+
+The staking miners are required to pay a deposit in order to post their solutions. The deposit is a
+sum of the `SignedDepositBase`, which is a fixed amount and `SignedDepositByte`, a variable amount
+per KB of solution data. All good solutions are subject to recieving a `SignedRewardBase`.
+
+Current deposit(**SignedDepositBase**) is
+{{ polkadot: <RPC network="polkadot" path="consts.electionProviderMultiPhase.signedDepositBase" defaultValue={16} filter="humanReadable"/> :polkadot }}
+{{ kusama: <RPC network="kusama" path="consts.electionProviderMultiPhase.signedDepositBase" defaultValue={16} filter="humanReadable"/> :kusama }}
+
+Current deposit per byte(**SignedDepositByte**) is
+{{ polkadot: <RPC network="polkadot" path="consts.electionProviderMultiPhase.signedDepositByte" defaultValue={16} filter="humanReadable"/> :polkadot }}
+{{ kusama: <RPC network="kusama" path="consts.electionProviderMultiPhase.signedDepositByte" defaultValue={16} filter="humanReadable"/> :kusama }}
+
+Current rewards(**SignedRewardBase**) is
+{{ polkadot: <RPC network="polkadot" path="consts.electionProviderMultiPhase.signedRewardBase" defaultValue={16} filter="humanReadable"/> :polkadot }}
+{{ kusama: <RPC network="kusama" path="consts.electionProviderMultiPhase.signedRewardBase" defaultValue={16} filter="humanReadable"/> :kusama }}
+
+Once the staking miner with a good solution is ready to be rewarded,
+[`finalize_signed_phase_accept_solution`](https://github.com/paritytech/substrate/blob/master/frame/election-provider-multi-phase/src/signed.rs#L453-L474)
+will be called, which will submit the solution to the queue of solutions, reward the staking miner
+and refund the deposit.
```

Comment:
Just a question that might come up from the readers -

It is unclear why the good solution is submitted to the queue of solutions here. What does this queue contain? Why can't the best solution that is being rewarded be directly accepted?

```suggestion
Once the staking miner with a good solution is ready to be rewarded,
[`finalize_signed_phase_accept_solution`](https://github.com/paritytech/substrate/blob/master/frame/election-provider-multi-phase/src/signed.rs#L453-L474)
will be called, which will submit the solution to the queue of solutions, reward the staking miner
and refund the deposit.
```

Reply:
Yeah, good point. It's well explained at the bottom of the page, I'll link.